pull_request_template.md

```diff
@@ -2,30 +2,16 @@
 
 _insert reason_
 
-Ticket: _link/ref to ticket/jira/cherwell/trello_
-
-Akamai ticket: _Cherwell ticket #_
+Related links: _link/ref to ticket/jira/cherwell/trello_
 
 ## Description of Changes
 
 _insert description_
 
-This is a:
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] Refactor (non-breaking change)
-- [ ] New or improved tests (non-breaking change)
-- [ ] New feature (non-breaking change which adds functionality or implements non-functional requirement)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
-- [ ] Documentation or design proposal
-
-
 ## Checklist
 
-- [ ] This PR has a descriptive title
-- [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation and docs have been updated.
```

Comment:
NIT: Can we change the phrasing of this to something like 'Documentation has updated where required.`
I find the conditional change comment confusing and I also dislike the use of the word my